bindings/matrix-sdk-ffi/src/platform.rs

```diff
@@ -108,6 +111,19 @@ fn setup_tracing_helper(configuration: String, colors: bool) {
         .init();
 }
 
+#[cfg(target_os = "android")]
+pub fn setup_tracing(configuration: String) {
```

Comment:
Is this one still used? Doesn't seem so?

Reply:
Oh, this must be `setup_tracing_impl`. Looks like CI has an Android-shaped hole.